myRental/scripts/setup.sh

```diff
@@ -2,7 +2,27 @@
 echo "Setting up NPM packages..."
 sudo npm install knex@0.13 --save
 sudo npm install bookshelf --save
-sudo npm install bookshelf-check-duplicates
+sudo npm install bookshelf-check-duplicates -g
```

Comment:
Do you know if this script works on Windows? I haven't tested it, but I know that Windows doesn't always like `sudo`

Reply:
Not sure, I know very little about Windows bash compatibility.